src/pages/u/[handle].tsx

```diff
@@ -4,14 +4,51 @@ import withAccessType from '@/lib/hoc/withAccessType';
 import { CookieService, PermissionService } from '@/lib/services';
 import type { PublicProfile } from '@/lib/types/apiResponses';
 import { CookieType } from '@/lib/types/enums';
+import defaultProfilePicture from '@/public/assets/profile-pics/adorable0.png';
+import styles from '@/styles/pages/u.module.scss';
+import Image from 'next/image';
 import type { GetServerSideProps } from 'next/types';
 
 interface UserProfilePageProps {
   user: PublicProfile;
 }
 
 const UserProfilePage = ({ user }: UserProfilePageProps) => {
-  return <pre>{JSON.stringify(user, null, 2)}</pre>;
+  // return <pre>{JSON.stringify(user, null, 2)}</pre>;
+  return (
+    <div className={styles.profilePage}>
+      <div className={styles.cardWrapper}>
+        <div className={styles.banner} />
+        <div className={styles.profileCard}>
+          <div className={styles.cardHalf}>
+            <div className={styles.horizontal}>
+              <Image
+                src={user.profilePicture ?? defaultProfilePicture}
+                alt="Profile Picture"
+                className={styles.profilePic}
+              />
+              <div style={{ marginLeft: '1rem' }}>
+                <h1>{`${user.firstName} ${user.lastName}`}</h1>
+                <h3>@{user.handle}</h3>
+                <h2>Rank Here</h2>
+              </div>
+            </div>
+            <div className={styles.iconBox}>icon icon icon icon</div>
```

Comment:
message me if you need help getting data to test the social media links with. i know we haven't implemented a way to add a link to your profile yet